### PR TITLE
Copy certificates for podman too

### PR DESCRIPTION
**Pull request by inercia**
## What does this PR change?

Copy the certificates for podman too.

## Documentation

No documentation needed

## Tests

* Add a new registry, as described in the quick start example in https://github.com/kubic-project/registries-operator.
* The `ca.crt` should be copied to `/etc/containers/certs.d/registry.suse.de:5000/ca.crt` in all the machines in the cluster.

## Links

See https://trello.com/c/77GLHQZx

### CODE REVIEW

File: pkg/controller/registry/registry_controller.go [hunk 1]
```diff
@@ -137,21 +141,23 @@ type ReconcileRegistry struct {
...
 // +kubebuilder:rbac:groups=kubic.opensuse.org,resources=registries,verbs=get;list;watch;create;update;patch;delete
 func (r *ReconcileRegistry) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	glog.V(5).Infof("[kubic] trying to reconcile %s", request.NamespacedName)
+	glog.V(5).Infof("[kubic] trying to reconcile registry %s", request.Name)
 
 	// Fetch the Registry instance
 	registry := &kubicv1beta1.Registry{}
 	ctx := context.Background()
-	if err := r.Get(ctx, request.NamespacedName, registry); err != nil {
+	// registries are not namespaced, so use only the Name as the search key
```
Comment (MalloZup) [inline pkg/controller/registry/registry_controller.go]: nice comment

File: build/make/runner.mk [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+# Usage:
+# - Run it locally:
+#   make local-run VERBOSE_LEVEL=5
+# - Start a Deployment with the manager:
+#   make local-run EXTRA_ARGS="--"
+#
+local-run: $(REGS_OPER_EXE) $(KUBECONFIG)
+	@echo ">>> Loading stuff with kubectl apply"
+	@for f in config/sas/*.yaml config/crds/*.yaml ; do $(KUBECTL) apply -f $$f ; done
+	@sleep 5
```
Comment (MalloZup) [inline build/make/runner.mk]: why we need to sleep here? we might comment this why we sleep for which event we wait. I think is ok to sleep, although i don't sleep enough in normal life :o:

Comment (MalloZup) [reply build/make/runner.mk]: i think sleep 5 could be ok

File: build/make/runner.mk [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+# assuming the k8s cluster is accessed with $(KUBECONFIG),
+# deploy the registries-operator manifest file in this cluster.
+local-deploy: $(REGS_DEPLOY) $(KUBECONFIG)
+	@echo ">>> (Re)deploying..."
+	@echo ">>> Deleting any previous resources..."
+	-@$(KUBECTL) get ldapconnectors -o jsonpath="{..metadata.name}" | \
+		xargs -r $(KUBECTL) delete --all=true ldapconnector 2>/dev/null
+	-@$(KUBECTL) get dexconfigurations -o jsonpath="{..metadata.name}" | \
+		xargs -r $(KUBECTL) delete --all=true dexconfiguration 2>/dev/null
+	@sleep 30
```
Comment (MalloZup) [inline build/make/runner.mk]: for this sleep,  we might create an event_function, would be worth for it?

File: Makefile [hunk 4]
```diff
@@ -112,93 +90,35 @@ clean: docker-image-clean
...
  #############################################################
 
-kustomize-deps:
-ifndef KUSTOMIZE_EXE
-	@echo ">>> kustomize does not seem to be installed. installing kustomize..."
-	$(GO) get -u sigs.k8s.io/kustomize
-endif
+$(KUSTOMIZE):
+	@echo ">>> Getting kustomize..."
+	-@$(GO_NOMOD) get -u sigs.k8s.io/kustomize
```
[the remark this answers is not in the document]
Comment (inercia) [reply Makefile]: `kustomize` can perform replacements in YAML files. It is a tweak used by kubebuilder for customizing things...

File: pkg/controller/registry/registry_cert_installer.go [hunk 5]
```diff
@@ -196,7 +205,8 @@ func (r *ReconcileRegistry) installCertForRegistry(registry *kubicv1beta1.Regist
 			jobInstallLabelHash:     getSecretHash(secret),
 		},
 		HostPaths: []string{
-			"/etc/docker", // mount this directory, so we can copy to the "certs.d" subdir
+			"/etc/docker",
```
[the remark this answers is not in the document]
Comment (inercia) [reply pkg/controller/registry/registry_cert_installer.go]: That comment doesn't apply anymore...

File: pkg/controller/registry/registry_controller.go [hunk 6]
```diff
@@ -43,8 +44,11 @@ const (
 	// the registries controller name
 	regsControllerName = "KubicRegistriesController"
 
-	// certificates directory
+	// certificates directory for Docker
 	dockerCertsDir = "/etc/docker/certs.d/"
+
+	// certificates directory for podman
```
[the remark this answers is not in the document]
Comment (inercia) [reply pkg/controller/registry/registry_controller.go]: Not sure, maybe...

File: build/make/runner.mk (hunk 2 above)
Comment (inercia) [reply build/make/runner.mk]: All these `kubectl` can take some time until the stuff they load is really loaded.

File: build/make/runner.mk (hunk 3 above)
Comment (inercia) [reply build/make/runner.mk]: I wouldn't waste so much time just for this makefile target...

File: Makefile (hunk 4 above)
[the remark this answers is not in the document]
Comment (MalloZup) [reply Makefile]: :lollipop:

**Pull request merged**